config/manifests/bases/clusterlogging.clusterserviceversion.yaml

```diff
@@ -241,11 +241,6 @@ spec:
         path: visualization.tolerations
         x-descriptors:
         - urn:alm:descriptor:com.tectonic.ui:selector:core:v1:Toleration
-      statusDescriptors:
```

Comment:
This removes it out of the console I believe which means I expect its broken on the UI for fluent deployments

Reply:
without this change fix not work :(